app/data_models/questionnaire_store.py

```diff
@@ -35,10 +40,10 @@ def __init__(self, storage, version=None):
         if version is not None:
             self.version = version
 
-    def get_latest_version_number(self):
+    def get_latest_version_number(self) -> int:
         return self.LATEST_VERSION
 
-    def set_metadata(self, to_set):
+    def set_metadata(self, to_set: Mapping[str, Union[str, int, list]]) -> Any:
```

Comment:
It returns `self` so should be a questionnaire store instance.

Reply:
Changed to QuestionnaireStore, cheers!